src/backend/libpq/rangerrest.c

```diff
@@ -366,7 +365,7 @@ static size_t write_callback(char *contents, size_t size, size_t nitems,
 	if (curl->response.buffer == NULL)
 	{
 		/* out of memory! */
-		elog(WARNING, "not enough memory for Ranger response");
+		elog(WARNING, "cannot allocate memory for ranger response");
```

Comment:
Why do you modify the out of memory error message. I think the original one is more specified, and may be better.